packages/@webex/plugin-logger/src/logger.js

```diff
@@ -60,6 +60,11 @@ function walkAndFilter(object, visited = []) {
       if (patterns.containsEmails.test(object)) {
         return object.replace(patterns.containsEmails, '[REDACTED]');
       }
+      if (patterns.containsMTID.test(object)) {
+        const regex = /(MTID=)[^&$#]*/g;
```

Comment:
Why do not you use the same regexp for both test and replace? This RE is good for testing as well.